lib/factory_bot/decorator.rb

```diff
@@ -6,10 +6,14 @@ def initialize(component)
       @component = component
     end
 
-    def method_missing(name, *args, &block)
+    def method_missing(name, *args, &block) # rubocop:disable Style/MethodMissingSuper
       @component.send(name, *args, &block)
     end
 
+    def respond_to_missing?(name, include_private = false)
+      @component.respond_to?(name) || super
```

Comment:
`method_missing` is using `send`, rather than `public_send`, so I think we might need to use `@component.respond_to?(name, true)` here to account for any private or protected methods.

Reply:
I did wonder!